pkg/util/azureclient/azureclient.go

```diff
@@ -32,16 +34,36 @@ func addAcceptLanguages(acceptLanguages []string) autorest.PrepareDecorator {
 			if err != nil {
 				return r, err
 			}
-			if acceptLanguages != nil {
-				for _, language := range acceptLanguages {
-					r.Header.Add("Accept-Language", language)
-				}
+			for _, language := range acceptLanguages {
+				r.Header.Add("Accept-Language", language)
 			}
 			return r, nil
 		})
 	}
 }
 
+type loggingSender struct {
+	autorest.Sender
+}
+
+func (ls *loggingSender) Do(req *http.Request) (*http.Response, error) {
+	b, _ := httputil.DumpRequestOut(req, true)
+	os.Stdout.Write(b)
+	resp, err := ls.Sender.Do(req)
+	if resp != nil {
+		b, _ = httputil.DumpResponse(resp, true)
+		os.Stdout.Write(b)
+	}
+	return resp, err
+}
+
+func setupClient(client *autorest.Client, authorizer autorest.Authorizer, languages []string) {
+	client.Authorizer = authorizer
+	client.RequestInspector = addAcceptLanguages(languages)
+	client.PollingDelay = 10 * time.Second
+	// client.Sender = &loggingSender{client.Sender}
```

Comment:
delete or uncomment?

Reply:
I have also been meaning to plump this via TestConfig. Seems like a useful thing to be able to enable in tests.

Reply:
Not sure we want all this noise in tests too. I would e fine to keep it non-configurable for now as in most cases it will be local runs, which needs this one. We already have tone of configurables :/

Reply:
> Not sure we want all this noise in tests too. I would e fine to keep it non-configurable for now as in most cases it will be local runs, which needs this one. We already have tone of configurables :/

We should be able to redirect this output into its own file. OK for now

/lgtm